xenforo-scraper.py

```diff
@@ -232,23 +240,31 @@ def scrapepage(url):
         if args.debug:
             print("Saving to:", fullpath)
 
+        import pdb
         if os.path.exists(fullpath):
-            print(f"\x1b[KProgress: {count}/{len(files)} - Skipping {truncated}", end="\r")
+            filename, extension = os.path.splitext(fullpath)
+            counter = 1
+            while os.path.exists(fullpath):
+                if os.path.getsize(fullpath) == filesize:
+                    print(f"\n[KProgress: {count}/{len(files)} - Skipping {truncated}", end="\r")
+                    break
+                fullpath = f"{filename} + ({counter}){extension}"
+                counter += 1
             continue
 
         if (maxsize is None or (maxsize is not None and filesize <= maxsize)) and (
                 minsize is None or (minsize is not None and filesize >= minsize)):
             difference = time.time() - timestamp
             print(
-                f"\x1b[KSpeed: {bytesToShort(totaldownloaded / difference)}/s Progress: {count}/{len(files)}"
+                f"\n[KSpeed: {bytesToShort(totaldownloaded / difference)}/s Progress: {count}/{len(files)}"
```

Comment:
What terminal emulator are you using? `\x1b[K` is an [ANSI escape code](https://en.wikipedia.org/wiki/ANSI_escape_code) for control sequence introducer followed by erase line. This creates the effect of the new text overwriting the old text. The escape code should be supported for most terminals, this change makes each line to go on their own line with an extra "[K" at the beginning.

Reply:
It was debug feature, my bad. Anyway you can use newline symbol in debug mode, not `\x1b[K].